tests/cpp/test_resize.cpp

```diff
@@ -3434,14 +3492,12 @@ TEST_F(ResizeTest, PadVectorization) {
   ASSERT_EQ(pparams->vectorization_factor, 4)
       << "Unexpected factor of vectorization";
 
-  // Make sure tv1 is not vectorized, i.e., no loop IterDomains are vectorized.
+  // Make sure tv1/tv2 are vectorized, i.e., at least one loop IterDomain is
```

Comment:
Is `tv1` supposed to be vectorized or not?

Reply:
I don't see anything that could block it.

Reply:
they are both vectorized and are supposed to be so.

The comment was wrong and I fixed that.